# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../etcnet/Makefile-ln.patch
# Copyright (C) 2007 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- etcnet-0.9.2/contrib/Makefile.orig	2007-05-08 17:01:02.000000000 +0200
+++ etcnet-0.9.2/contrib/Makefile	2007-05-08 17:02:50.000000000 +0200
@@ -57,8 +57,8 @@
 
 # 3. Create /sbin links.
 	@install -d $(sbindir)
-	@ln -s ../etc/net/scripts/ifup $(sbindir)/ifup
-	@ln -s ../etc/net/scripts/ifdown $(sbindir)/ifdown
-	@ln -s ../etc/net/scripts/contrib/efw $(sbindir)/efw
-	@ln -s ../etc/net/scripts/contrib/eqos $(sbindir)/eqos
+	@ln -sf ../etc/net/scripts/ifup $(sbindir)/ifup
+	@ln -sf ../etc/net/scripts/ifdown $(sbindir)/ifdown
+	@ln -sf ../etc/net/scripts/contrib/efw $(sbindir)/efw
+	@ln -sf ../etc/net/scripts/contrib/eqos $(sbindir)/eqos
 	@echo Done!
